public/server/db/dfw.json

```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "Dead From Writting"
```

Comment:
Okay, so this is quite a new "slang" for me, and before I merge it (or not), is it okay to maybe provide some context where this has been used before? Maybe a phrase,or what it means in general, because I cannot seem to find it. I know that dfw in slang most probably abbreviated for "down for whatever"